ios/lockdown_assistivetouch.go

```diff
@@ -0,0 +1,52 @@
+package ios
+
+import log "github.com/sirupsen/logrus"
+import "fmt"
+
+const accessibilityDomain = "com.apple.Accessibility"
+const assistiveTouchKey  = "AssistiveTouchEnabledByiTunes"
+
+// EnableAssistiveTouch creates a new lockdown session for the device and enables or disables
+// AssistiveTouch (the on-screen software home button), by using the special key AssistiveTouchEnabledByiTunes.
+// Setting to true will enable AssistiveTouch
+// Setting to false will disable AssistiveTouch, regardless of whether it was previously-enabled through
+// a non-iTunes-related method.
+func SetAssistiveTouch(device DeviceEntry, enabled bool) error {
+	lockDownConn, err := ConnectLockdownWithSession(device)
+	if err != nil {
+		return err
+	}
+	log.Debugf("Setting %s: %t", assistiveTouchKey, enabled)
+	defer lockDownConn.Close()
+	err = lockDownConn.SetValueForDomain(assistiveTouchKey, accessibilityDomain, enabled)
+	return err
+}
+func GetAssistiveTouch(device DeviceEntry) (bool, error) {
+	lockDownConn, err := ConnectLockdownWithSession(device)
+
+	if err != nil {
+		return false, err
+	}
+	defer lockDownConn.Close()
+
+	enabledIntf, err := lockDownConn.GetValueForDomain(assistiveTouchKey, accessibilityDomain)
+
+	if err != nil {
+		return false, err
+	}else if enabledIntf == nil{
```

Comment:
please don't use else if. 
```
if ... {
return ..
}
if enabledIntf==nil{
..
}
```
Does the same with less nesting. Usually it is easier to read.